test/common/tcp_proxy/tcp_proxy_test.cc

```diff
@@ -1309,6 +1333,25 @@ TEST_F(TcpProxyRoutingTest, DEPRECATED_FEATURE_TEST(ApplicationProtocols)) {
   filter_->onNewConnection();
 }
 
+// Test TCP proxy use source IP to hash.
+TEST_F(TcpProxyRoutingTest, DEPRECATED_FEATURE_TEST(HashWithSourceIp)) {
```

Comment:
not sure what `DEPRECATED_FEATURE_TEST` is but I'm guessing you dont want to use that here?

Reply:
I have to use it here because `setup` uses deprecate field in the test configuration.

Reply:
ok, thanks for explaining